gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -61,16 +65,30 @@ public class AndroidCommandPluginExtension {
     // prefer system property over direct setting to enable commandline arguments
     @Memoized
     def getDeviceId() {
-        if (System.properties['deviceId'])
+        if (System.properties['deviceId']) {
             return System.properties['deviceId']
-        if (deviceId instanceof Closure)
+        } else if (deviceId instanceof Closure) {
             return deviceId.call()
+        }
         deviceId ?: defaultDeviceId()
     }
 
-    // prefer system property over direct setting to enable commandline arguments
     def getEvents() {
-        System.properties['events'] ?: events ?: 10000
+        if (System.properties['events']) {
+            return System.properties['events']
+        } else if (events) {
+            return events
+        }
+        EVENTS_DEFAULT
+    }
```

Comment:
Yes, that's true. Would make things a bit easier. 
I'm sorry that this now affects your contribution, but I have to make sure that these opinionated changes don't sneak and cause a lot of discussion when changing them later on. :smile: 

I'll give you two :+1: for reverting. :dancers: